Else setuptools installs an .egg directory, this way it installs
nftables/ and nftables-*-egg.info/

--- a/py/Makefile.am
+++ b/py/Makefile.am
@@ -7,7 +7,7 @@
 install-exec-local:
 	cd $(srcdir) && \
 		$(PYTHON_BIN) setup.py build --build-base $(abs_builddir) \
-		install --prefix $(DESTDIR)$(prefix)
+		install --prefix $(prefix) --root $(DESTDIR)
 
 uninstall-local:
 	rm -rf $(DESTDIR)$(prefix)/lib*/python*/site-packages/nftables
